### ISSUE TITLE
Bad finetune accuracy .

### ISSUE THREAD

**Issue by Victoria2333** [davidsandberg/facenet#828]
I followed issue #139. Here is my specific step:
Here is my code:
all_vars = tf.trainable_variables()
var_to_restore = [v for v in all_vars if not v.name.startswith('Mixed_8b')] in inception-renet-v1
saver = tf.train.Saver(var_to_restore)
I run:
python src/train_softmax.py --data_dir /home/han/facenet/train_face_160 --image_size 160 __embedding_size 512 /home/han/facenet/src/pretrained_models/20180408-102900/model-2018408-102900.ckpt-90
I got a ckpt file(for example, 1.ckpt), i only trained for 4 epoch,coz loss is nearly 0 at 1 epoch...
And I freeze the created 1.ckpt file to 1.pb file which is loaded in my_classifier.py to train and test but the training accuracy is 1 while test accuracy is about 0.4...
I don't know why finetuned model is worse than the pretrained-model(model-2018408-102900.ckpt-90)

Second, I still don't know if following my code, if the layers except 'Mixed_8b' can be trained?? and this layer will be droped?? Am I right??? Is the last layer 'Logits' dropped?? I've tried var_to_restore = [v for v in all_vars if not (v.name.startswith('Mixed_8b') or v.name.startswith('Logits'))] and it all got 0.3~0.4. Theoretically, the result should be better than unfintuned model?? The unfinetuned result is 0.803.

I want to, for exmaple, freeze the old weights of L-2 layer and restore the weights of remaining 2 layer and finetune the last 2 layers. I don't mean to drop some variables which could lower the accuracy, so how can i finetune??? I think maybe #139 method is not right......
Hope someone could help me.

Victoria

**Comment by taureanamir** [comment]
@Victoria2333, Did you find a way to fine tune??
